conf/messages.en

```diff
@@ -131,28 +131,29 @@ soleTraderName.error.lastName.required = Enter your last name
 soleTraderName.error.firstName.length = First name must be 35 characters or less
 soleTraderName.error.lastName.length = Last name must be 35 characters or less
 
-utrCorporationTax.title =
-utrCorporationTax.heading =
-utrCorporationTax.p1 =
-utrCorporationTax.link.text =
-utrCorporationTax.link.href =
-
-utrPartnership.title =
-utrPartnership.heading =
-utrPartnership.p1 =
-utrPartnership.link.text =
-utrPartnership.link.href =
-
-utrSelfAssessment.title =
-utrSelfAssessment.heading =
-utrSelfAssessment.p1 =
-utrSelfAssessment.link.text =
-utrSelfAssessment.link.href =
-
-utr.checkYourAnswersLabel = utr
-utr.error.required = Enter utr
-utr.error.length = Utr must be 100 characters or less
-utr.change.hidden = Utr
+utrCorporationTax.title = What is your Corporation Tax Unique Taxpayer Reference (UTR)?
+utrCorporationTax.heading = What is your Corporation Tax Unique Taxpayer Reference (UTR)?
+utrCorporationTax.p1 = Your UTR can be 10 or 13 digits long. It will be on tax returns and other letters about Corporation Tax. It may be called 'reference', 'UTR' or 'official user'.
```

Comment:
We need to use "curly quotes" - i.e. ’ and ’ - rather than straight apostrophes in content - you should be able to copy them from the "hasUtr" content

Reply:
Changed but TBH I don't see the difference!